coala_quickstart/green_mode/green_mode.py

```diff
@@ -425,3 +435,167 @@ def bear_test_fun(bears, bear_settings_obj, file_dict, ignore_ranges,
             final_unified_results.append(unified_file_results)
 
     return final_non_op_results, final_unified_results
+
+
+def generate_data_struct_for_sections(results):
```

Comment:
the name is already long :p let's keep it `generate_data_structure_for_sections` only